### PR TITLE
README.markdown install from repos

### PR DESCRIPTION
**Pull request by stokito**
Also added a link to Awesome Selfhosted list that has many other MTAs.

Please add tags to the gitub repo to make it easier to find:  smtp smtp-server mta mail-server

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -24,14 +31,19 @@ In BSD:
 	cd bsd && make
 
 Installation
-------------
```
Comment (corecode) [inline README.markdown:27]: why?

Comment (stokito) [reply README.markdown:27]: It's build built and installation from sources. For me it looks like a just a build step. I can revert

**Comment by corecode** [comment]
I'd say keep this and call the new section something like "Install from 
package managers"?

**Comment by stokito** [comment]
I updated the branch

**Pull request merged**